install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```
@@ -283,6 +283,7 @@ spec:
             - NET_ADMIN
             - SYS_MODULE
           privileged: true
+          {{- toYaml .Values.securityContext | trim | nindent 10 }}
```

GitHub action fails with `Error: YAML parse error on cilium/templates/cilium-agent-daemonset.yaml: error converting YAML to JSON: yaml: line 144: did not find expected key`. I suspect this is because `.Values.securityContext` is empty by default. Maybe try to guard it like this?:

```suggestion
{{- if .Values.securityContext }}
          {{- toYaml .Values.securityContext | trim | nindent 10 }}
{{- end }}
```